src/main/java/com/powsybl/openloadflow/OpenLoadFlowParameters.java

```diff
@@ -16,36 +18,38 @@
 import java.util.List;
 import java.util.Objects;
 
+import static com.powsybl.openloadflow.util.ParameterConstants.*;
+
 /**
  * @author Geoffroy Jamgotchian <geoffroy.jamgotchian at rte-france.com>
  */
 public class OpenLoadFlowParameters extends AbstractExtension<LoadFlowParameters> {
 
     private SlackBusSelector slackBusSelector = new MostMeshedSlackBusSelector();
 
-    private boolean distributedSlack = true;
+    private boolean distributedSlack = DISTRIBUTED_SLACK_DEFAULT_VALUE;
+
+    private boolean throwsExceptionInCaseOfSlackDistributionFailure = THROWS_EXCEPTION_IN_CASE_OF_SLACK_DISTRIBUTION_FAILURE_DEFAULT_VALUE;
+
+    private BalanceType balanceType = BALANCE_TYPE_DEFAULT_VALUE;
+
+    private boolean dc = DC_DEFAULT_VALUE;
 
-    private boolean throwsExceptionInCaseOfSlackDistributionFailure = true;
+    private boolean voltageRemoteControl = VOLTAGE_REMOTE_CONTROLE_DEFAULT_VALUE;
 
-    private BalanceType balanceType = BalanceType.PROPORTIONAL_TO_GENERATION_P_MAX; // Default slack distribution on generators.
+    private LowImpedanceBranchMode lowImpedanceBranchMode = LOW_IMPEDANCE_BRANCH_MODE_DEFAULT_VALUE;
 
     public enum BalanceType {
         PROPORTIONAL_TO_GENERATION_P, // Not implemented yet.
         PROPORTIONAL_TO_GENERATION_P_MAX,
         PROPORTIONAL_TO_LOAD,
```

Comment:
Can you add ````PROPORTIONAL_TO_CONFORM_LOAD```` ?

Reply:
Added with an exception thown like for "PROPORTIONAL_TO_GENERATION_P"
implementation should come with the issue #110

Reply:
It is implemented indeed, see in the master branch. The issue is partly implemented.